metadata/rest/test/test_orm.py

```diff
@@ -126,7 +126,7 @@ def test_perf_of_large_insert(self):
                                data=txt_trans, headers=self.headers)
             self.assertEqual(req.status_code, 200)
         end_time = time()
-        self.assertTrue(end_time - start_time < 90)
```

Comment:
This should be reverted... or we should document justification for changing it

Reply:
Rolling this one back. This test was failing multiple times, but I think it was just Travis misbehaving.